api/features/tests/test_views.py

```diff
@@ -646,3 +647,23 @@ def test_get_flags_exclude_disabled(self):
 
         # but enabled ones are
         assert response_json[0]["feature"]["id"] == enabled_flag.id
+
+
+@pytest.mark.parametrize(
+    "client", [(lazy_fixture("master_api_key_client")), (lazy_fixture("admin_client"))]
+)
+def test_get_feature_states_by_uuid(client, environment, feature, feature_state):
+    # Given
+    base_url = reverse(
+        "api-v1:features:get-feature-state-by-uuid", args=[feature_state.uuid]
+    )
+    url = f"{base_url}?environment={environment}"
```

Comment:
If we're getting the feature state by uuid I'm a bit unclear why we need the environment query parameter here?